### DISCUSSION TITLE
突然无法科学了

### DISCUSSION THREAD

**Discussion by hao-888** [kkkgo/PaoPaoGateWay#78] [问和答]
![17113372071887082122565158587358](https://github.com/kkkgo/PaoPaoGateWay/assets/64783863/00ed2aa7-0da7-4946-a78a-dfd6775e6179)

**Comment by hao-888** [comment]
![17113373357167815184093653515942](https://github.com/kkkgo/PaoPaoGateWay/assets/64783863/5c01ca82-7739-4765-bbf0-b694a2af24ac)

**Comment by hao-888** [comment]
yacd 那边，所有节点都无法测速

**Comment by kkkgo** [comment]
我觉得是和你的规则配置有问题，很明显所有流量都走了直连

**Reply by kkkgo** [reply]
仔细你看不是走了直连，而是你自己的节点连不上了，是你节点不行。

**Reply by kkkgo** [reply]
gz***.com:55570 connection refused，不用解释了吧。下次建议换个客户端测试排除下。

**Comment by hao-888** [comment]
抱歉，错误有点低级😢，另外问一下，网关和 dns，不出问题的情况下，需要时间久了升级一下吗

**Reply by kkkgo** [reply]
你可以关注一下DNS的更新日志和网关的release更新说明。

**Reply by kkkgo** [reply]
[PaoPao DNS更新日志](https://github.com/kkkgo/PaoPaoDNS/discussions/categories/%E6%9B%B4%E6%96%B0%E6%97%A5%E5%BF%97)
 [PaoPao Gateway releases ](https://github.com/kkkgo/PaoPaoGateWay/releases)
